packages/edge/src/commands/edge/drivers/default.ts

```diff
@@ -15,16 +15,31 @@ export default class DriversDefaultCommand extends EdgeCommand<typeof DriversDef
 		...outputList.flags,
 	}
 
+	static args = [{
+		name: 'idOrIndex',
+		description: 'the driver id or number in list',
+	}]
+
 	static examples = [`# list default drivers
 $ smartthings edge:drivers:default`]
```

Comment:
As long as this command is being updated, may as well update the example to the current format:
`static examples = [
		{ description: 'list default drivers', command: 'smartthings edge:drivers:default' }
]`